.github/workflows/macos-release-candidate.yml

```diff
@@ -89,7 +110,7 @@ jobs:
     - name: Upload release artifacts to Google Cloud Storage bucket
       run: |
         gsutil rm -rf ${{ env.BUCKET_PATH }} || true
-        gsutil cp -r dist/installers ${{ env.BUCKET_PATH }}
+        gsutil cp dist/installers/*.dmg* dist/installers/latest-mac.yml ${{ env.BUCKET_PATH }}
```

Comment:
Much like I observed above for Linux, `cp -r` no longer works because of the contents of the "unpacked" directory. For macOS we're expecting:

1.  The `.dmg` installer
2.  The `.dmg.blockmap` file (described at electron-userland/electron-builder#2851 as being involved with "partial/differential updates to your application, to shrink the update payload size")
3.  File `latest-mac.yml` (related to auto-update, described somewhat at https://www.electron.build/auto-update)